src/pages/Components/RatingStars/code.md

```diff
@@ -3,39 +3,198 @@ title: 'Code'
 order: 2
 ---
 
-## Import
+> Rating stars can be used in two ways : (1.) when filing or amending an opinion (2.) when displaying the overall score on a product sheet.
+
+Implementing rating stars :
+
+- [When of filing or amending an opinion](#case-1)
+- [When displaying the overall score](#case-2)
+
+## Filing or amending an opinion
+
+<a name="case-1"></a>
+
+### Import
+
+Import the settings and the stars `scss` files
 
 ```css
 @import 'settings-tools/_all-settings';
-@import '...';
+@import 'components/_c.stars-input';
 ```
 
-## Basic usage
+### Basic usage
 
-Apply the classes...
+To use the rating stars when filing or modifying a opinion, you must apply the `mc-stars-input` class to an html element (div or fieldset) used as a container for input radio tags (radio buttons). These radio buttons must be associated with label tags.
 
 ```html
-<div class="class">Label</div>
+<div class="mc-stars-input">
+  <input type="radio" name="rating" value="0" id="no-rate" class="sr-only mc-stars-input__radio" checked aria-label="No opinion given" />
+
+  <input type="radio" name="rating" value="1" id="ratingstars1" class="sr-only mc-stars-input__radio">
+  <label for="ratingstars1" class="mc-stars-input__label">
+    <span class="sr-only mc-stars-input__text">Very bad</span>
+    <svg class="mc-stars-input__icon"><use xlink:href="#starFullS" /></svg>
```

Comment:
I think we can use background and aria-labels and aria-hidden to reduce Dom footprint : 
```html
<input type="radio" name="rating" value="2" id="ratingstars2" class="sr-only mc-stars-input__radio" aria-lebel="Bad (2/5)">
<label for="ratingstars2" aria-hidden class="mc-stars-input__label"></label>
```

and we avoid the need for importing and having the SVGs as dependency